### ISSUE TITLE
Conv2d and BatchNorm should support optional fused activation function

### ISSUE THREAD

**Issue by wchao1115** [webmachinelearning/webnn#185]
When webnn is used as a target backend of frameworks, it is likely given a convolution or batchnorm operation together with a fused `relu` or other activation function that are widely known fusions supported in most frameworks today including ONNX Runtime and TensorFlow.js. Here is the compiled list of known activation functions that can be specified as a fused activation to these two operations.

``` javascript
enum MLFusedActivation {
  "elu",
  "hard-sigmoid",
  "leaky-relu"
  "linear",
  "relu",
  "selu",
  "sigmoid",
  "softplus",
  "softsign",
  "tanh"
};
```

**Comment by huningxin** [comment]
Thanks @wchao1115 . According to my investigations, the frameworks also usually fuse the bias into conv2d. It would be good if webnn supports the fused bias in conv2d op.

WDYT?

**Comment by huningxin** [comment]
Regarding to the `MLFusedActivation`, should it include the `clamp`? It is useful to support `relu6` of MobileNet.

**Comment by wchao1115** [comment]
Sounds good. I'll add `clamp` and `relu6` in the list as well. Regarding fused bias, can you give an example of the backend API that supports that? Note that frameworks can fuse anything at high level but without a broad backend support, the implementation will be forced to decouple the fused operation anyway. So I think we should consider fused operation first in the context of platform support.

**Comment by wchao1115** [comment]
BTW I'm going to avoid the never-ending debate by not asking "why 6?" ;-)

**Comment by huningxin** [comment]
> Regarding fused bias, can you give an example of the backend API that supports that?

A couple of examples:
 * NNAPI's [`ANEURALNETWORKS_CONV_2D`](https://developer.android.com/ndk/reference/group/neural-networks#group___neural_networks_1ggaabbe492c60331b13038e39d4207940e0a34a73b5eaf458b67db5eda71557d1d01) supports fused bias via the third operand, "2: A 1-D tensor, of shape [depth_out], specifying the bias".
 * DirectML's [`DML_CONVOLUTION_OPERATOR_DESC`](https://docs.microsoft.com/en-us/windows/win32/api/directml/ns-directml-dml_convolution_operator_desc) supports fused bias via the member `const DML_TENSOR_DESC     *BiasTensor;`.
 * MLCompute's [`MLCConvolutionLayer`](https://developer.apple.com/documentation/mlcompute/mlcconvolutionlayer) supports fused bias via `biases: MLCTensor?` parameter.
 * oneDNN's [`convolution`](https://oneapi-src.github.io/oneDNN/dev_guide_convolution.html) also supports bias via

**Comment by huningxin** [comment]
Cross refer to the discussion in the Chromium [CL](https://chromium-review.googlesource.com/c/chromium/src/+/4031876/comment/ca0cf6c4_bedec122/) review about whether WebNN should support fused operators.

/cc @wacky6
